htdocs/core/class/translate.class.php

```diff
@@ -170,9 +170,17 @@ public function getDefaultLang($mode = 0)
 	 */
 	public function loadLangs($domains)
 	{
+		$loaded = 0;
 		foreach ($domains as $domain) {
-			$this->load($domain);
+			$result = $this->load($domain);
+			if( $result>0 ){
```

Comment:
No space before the )
Need a space between the if and (
If you add the git hook as explained here https://wiki.dolibarr.org/index.php?title=Outils_de_d%C3%A9veloppement#Installer_le_hook_git_Pre-commit
such error will be fixed automatically